packages/orchestration/src/chain-info.js

```diff
@@ -1,101 +1,77 @@
-// UNTIL https://github.com/Agoric/agoric-sdk/issues/8879
-
-/** @file temporary static lookup of chain info */
-
 import { registerChain } from './utils/chainHub.js';
 
+// Refresh with scripts/refresh-chain-info.ts
+import fetchedChainInfo from './fetched-chain-info.js';
+
 /** @import {CosmosChainInfo, EthChainInfo} from './types.js'; */
 
 /** @typedef {CosmosChainInfo | EthChainInfo} ChainInfo */
 
-// TODO generate this automatically with a build script drawing on data sources such as https://github.com/cosmos/chain-registry
-
-// XXX inlines the enum values to save the import (entraining cosmic-proto which is megabytes)
-
-export const wellKnownChainInfo =
-  /** @satisfies {Record<string, ChainInfo>} */ (
-    harden({
-      agoric: {
-        chainId: 'agoriclocal',
-        connections: {
-          cosmoslocal: {
-            id: 'connection-1',
-            client_id: '07-tendermint-3',
-            counterparty: {
-              client_id: '07-tendermint-2',
-              connection_id: 'connection-1',
-              prefix: {
-                key_prefix: '',
-              },
+const knownChains = /** @satisfies {Record<string, ChainInfo>} */ (
+  harden({
+    ...fetchedChainInfo,
```

Comment:
the fetchedChainInfo seems to be mainnet info; should the agoric chainId change to `agoric-3` likewise?

How does it work with local starship-style testing? or in an instagoric-style test network?